instance/scripts/check_ssl_expiration.js

```diff
@@ -0,0 +1,52 @@
+var request = require('request');  
+
+/* 
+************************************************** 
+What is the URL that you want to check? 
+************************************************** 
+*/ 
+var urlToMonitor = 'https://example.com'; 
```

Comment:
Can you add a comment here to explain that this url is replaced programmatically?

Reply:
:+1: